action.yml

```diff
@@ -86,6 +86,10 @@ inputs:
     description: Post plans as one comment
     required: false
     default: 'false'
+  reporting-strategy:
+    description: 'comments_per_run or latest_run_comment, anything else will default to original behavior of multiple comments'
```

Comment:
@Spartakovic I think that the default behaviour should be `comments_per_run` since multiple_comments is too noisy for all people

Reply:
lets change it, there is already a manual version update so there shouldn't be surprises I think, we can advise that in the release notes too
in many ways this is much better than the previous behaviour